backend-api/src/functions/services/events/types.ts

```diff
@@ -65,6 +66,7 @@ export interface CredentialTokenIssuedEventConfig extends BaseEventConfig {
 
 export interface BiometricTokenIssuedEventConfig extends BaseUserEventConfig {
   documentType: DocumentType;
+  extensions?: Extensions;
```

Comment:
```suggestion (non-blocking)
  extensions: Extensions | undefined;
```

With this event specific event we always pass through the redirect uri we just don't know if it's defined or not, so would it make more sense to have this as mandatory to pass it through

Just a thought, happy if you've got a better reason not to do this

Reply:
See comment re passing extensions through in general as well, we could just pass redirectUri here instead of extensions

Reply:
I have it within the object itself to simplify the eventService itsels and to have an easier type system for extensions

Reply:
Updated this